app/controllers/jitsi_calls_controller.rb

```diff
@@ -0,0 +1,22 @@
+class JitsiCallsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    room_name = SecureRandom.uuid
+    @jitsi_call = JitsiCall.new({ room_name: room_name })
+    if @jitsi_call.save && User.exists?(call_params[:user_id])
+      assign_participants
+    else redirect_to root_path, alert: I18n.t('denial.resource_creation', resource: JitsiCall) end
```

Comment:
I'm very confused that RuboCop does not forbid `else ... end` in one line :sweat_smile:

Reply:
I removed the `end`